src/pocketmine/plugin/RegisteredListener.php

```diff
@@ -91,7 +90,7 @@ public function getPriority() : int{
 	 * @param Event $event
 	 */
 	public function callEvent(Event $event){
-		if($event instanceof Cancellable and $event->isCancelled() and $this->isIgnoringCancelled()){
+		if($event->isCancellable() and $event->isCancelled() and $this->isIgnoringCancelled()){
```

Comment:
This call looks quite redundant. Why not always return false for non-cancellable `Event::isCancelled()` calls, rather than throwing exceptions?